dashboard/public/l10n/locales.ini

```diff
@@ -25,6 +25,7 @@ viewAllClassrooms=View all classrooms
 allClassrooms=All Classrooms
```

Comment:
> If you cannot add translation for some language, add placeholder English translation instead so that it can be translated later.

can you explain this more?